tools/docker/Dockerfile.integ-test

```diff
@@ -30,15 +30,29 @@ RUN mkdir -p \
   ${FICD_LOG_DIR} \
   /etc/cni/net.d
 
+RUN echo `uname -m`
+
 # Install containerd to have runc shim.
-ENV CTRD_VERSION="1.6.4"
-RUN wget --quiet -O- https://github.com/containerd/containerd/releases/download/v$CTRD_VERSION/containerd-$CTRD_VERSION-linux-amd64.tar.gz | tar zxf - -C /tmp/ && \
+ENV CTRD_VERSION="1.6.8"
+RUN ARCH=`uname -m` && \
```

Comment:
I believe that you can call `go` command in this Dockerfile. Can you try `go env GOARCH`? That should return amd64/arm64 instead.

Reply:
After trying `go env GOARCH`, it returned arm64

Reply:
Added ` go env GOARCH` for CTRD and critest.